### PR TITLE
feat(route): add fisher spb news

### PR DESCRIPTION
**Pull request by denis-ya**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/fisher-spb/news
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [ ] Documentation
  - [ ] CN
  - [x] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by TonyRL** [comment]
Please use `[x]` for the checklist.

File: lib/v2/fisher-spb/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'fisher.spb.ru': {
+        _name: 'fisher spb',
+        '.': [
+            {
+                title: 'news',
+                docs: 'https://docs.rsshub.app/en/other.html#fisher-spb',
+                source: ['/news'],
+                target: '/news',
```
Comment (TonyRL) [inline lib/v2/fisher-spb/radar.js:9]:
```suggestion
                target: '/fisher-spb/news',
```

File: lib/v2/fisher-spb/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/news': ['denis.ya'],
```
Comment (TonyRL) [inline lib/v2/fisher-spb/maintainer.js:2]:
```suggestion
    '/news': ['denis-ya'],
```

File: docs/en/other.md [hunk 3]
```diff
@@ -220,3 +220,9 @@ Type
 | all | rec       |
 
 </RouteEn>
+
+## Fisher Spb
+
+### News
+
+<Route author="denis.ya" example="/news" path="/news" />
```
Comment (TonyRL) [inline docs/en/other.md:228]:
```suggestion
<RouteEn author="denis-ya" example="/fisher-spb/news" path="/fisher-spb/news" />
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by denis-ya** [comment]
> Please use `[x]` for the checklist.

fixed and suggestions applied as well

File: lib/v2/fisher-spb/news.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    });
+
+    const $ = cheerio.load(iconv.decode(response.data, 'windows-1251'));
+
+    const descBuilder = (element) => {
+        const content = cheerio.load(`<p>${$('.news-message-text', element).html()}</p>`).root();
+        $('.news-message-media a', element).each((_, elem) => {
+            if ($(elem).hasClass('news-message-youtube')) {
+                content.append(
+                    `<iframe type="text/html" width="640" height="360" src="https://youtube.com/embed/${$(elem)
```
Comment (NeverBehave) [inline lib/v2/fisher-spb/news.js:22]: Please use templates `art` for any html render

**Review by NeverBehave — CHANGES_REQUESTED** [review]
Let me know if anything unclear about the templates

**Comment by denis-ya** [comment]
> Let me know if anything unclear about the templates

Hello, it would be very helpfull to see some example with template as I have never used these before.

File: lib/v2/fisher-spb/news.js (hunk 4 above)
Comment (denis-ya) [reply lib/v2/fisher-spb/news.js]: done

**Review by NeverBehave — APPROVED** [review]
lgtm

**Comment by NeverBehave** [comment]
👍

**Pull request merged**